### PR TITLE
BP-4280 :  hin verification customisation

### PR DESCRIPTION
**Pull request by kelvinOBA**
This is a styling change to the verification-vue component, from text colours, background and button colour changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,12 +5,13 @@ dist
 # local env files
 .env.local
 .env.*.local
-
+.env
```
Comment (alnicoli) [inline .gitignore:8]: .env shouldn't be ignored. Only the local ones which are already in .gitignore

File: .gitignore [hunk 2]
```diff
@@ -5,12 +5,13 @@ dist
...
 # Log files
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 pnpm-debug.log*
+yarn.lock
```
Comment (alnicoli) [inline .gitignore:14]: Same here, shoulnd't be in .gitignore

File: packages/verification-vue-component/src/components/CertifactionVerification.vue [hunk 3]
```diff
@@ -28,9 +29,9 @@
 
         <div v-if="!digitalTwinModeActive" class="powered-by">
             <span class="label">{{ _$t('verification.poweredBy.label') }}</span>
-            <a href="https://certifaction.com" target="_blank">
-                <img src="../assets/img/certifaction_logo.svg" alt="Certifaction"/>
-            </a>
+                   <a href="https://certifaction.com" target="_blank">
+                        <img src="../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                   </a>
```
Comment (alnicoli) [inline packages/verification-vue-component/src/components/CertifactionVerification.vue:34]: Wrong code indentation

File: packages/verification-vue-component/src/components/Verification/VerificationItem.vue [hunk 4]
```diff
@@ -3,7 +3,12 @@
          :class="{ 'confirmation-step': digitalTwin.confirmationStep, 'error': digitalTwinInformation.error }">
         <div class="card-container">
             <div v-if="digitalTwinInformation.active" class="header">
-                <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                <template v-if="urlTypeLoaded === 'HIN'">
+                    <img src="../../assets/img/hin-logo.svg" alt="Hin"/>
+                </template>
+                <template v-else>
+                    <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                </template>
```
Comment (alnicoli) [inline packages/verification-vue-component/src/components/Verification/VerificationItem.vue:11]: This package is public which can be downloaded and used by anyone. I don't think we should have customer specific (HIN) code in here. ATM the logo is the only thing (beside the styles) which is custom, so I probably would pass it as prop to this package or maybe it's even possible to have it as a css variable (background image) as well. Then no additional prop would be needed and we have this clean only css variable solution

File: packages/verification-vue-component/src/style/components/_buttons.scss [hunk 5]
```diff
@@ -20,11 +20,12 @@
 }
 
 .btn-primary {
-    background-color: $verification-blue;
+    background-color: $btn-default-bg-color;
     color: $white;
+    border:  $btn-default-bg-color;
```
Comment (alnicoli) [inline packages/verification-vue-component/src/style/components/_buttons.scss:25]: formatting issue: two spaces

File: packages/verification-vue-component/src/style/components/certifaction_verification.scss [hunk 6]
```diff
@@ -41,6 +41,10 @@
         font-size: $font-size-small;
     }
 
+    a{
```
Comment (alnicoli) [inline packages/verification-vue-component/src/style/components/certifaction_verification.scss:44]: formatting issue: no space between 'a' and '{'

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 7]
```diff
@@ -1,3 +1,23 @@
...
+:root {
+    --cf-verification-primary-color: #0669f2;
+    --cf-verification-secondary-btn-bg-color: #e5e3d6;
+    --cf-verification-default-btn-bg-color: #ffffff ;
+    --cf-verification-action-box-bg-color: #e6f0fe ;
+    --cf-verification-action-box-text-color: #0d6efd ;
+}
+
+
+
```
Comment (alnicoli) [inline packages/verification-vue-component/src/style/settings/_variables.scss:11]: formatting issue: too many new lines

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 8]
```diff
@@ -1,3 +1,23 @@
...
+$btn-default-bg-color:var(#{--cf-verification-primary-color});
+$btn-secondary-bg-color:var(#{--cf-verification-secondary-btn-bg-color});
+$btn-secondary-text-color:var(#{--cf-verification-primary-color});
+$action-box-bg-color:var(#{--cf-verification-action-box-bg-color});
+$action-box-text-color:var(#{--cf-verification-action-box-text-color});
+$link-color:var(#{--cf-verification-primary-color});
+
+
+
```
Comment (alnicoli) [inline packages/verification-vue-component/src/style/settings/_variables.scss:20]: formatting issue: too many new lines

File: packages/verification-app/.env [hunk 9]
```diff
@@ -7,4 +7,4 @@ VUE_APP_CLAIM_CONTRACT_ADDRESS=
 VUE_APP_ACCEPTED_ISSUER_KEY=
 VUE_APP_CERTIFACTION_API_URL=
 VUE_APP_CERTIFACTION_CDN_BASE_URL=
-VUE_APP_PDF_WASM_VERSION=1.6.17
+VUE_APP_PDF_WASM_VERSION=1.6.22
```
Comment (alnicoli) [inline packages/verification-app/.env:10]: HIN multisign changes are behind v2.0.0+. So it would probably make sense to update it to the newest version but maybe there's a reason to not do it now. @bauer00 has more insight and I would wait also for his feedback

Comment (bauer00) [reply packages/verification-app/.env:10]: Yes, would make sense, but as it probably also needs changes in the wasm wrapper, worker and service, a separate branch would be better for that.

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 10]
```diff
@@ -1,3 +1,19 @@
...
+    --cf-verification-action-box-bg-color: #e6f0fe ;
+    --cf-verification-action-box-text-color: #0d6efd ;
+}
+
+$btn-default-bg-color:var(#{--cf-verification-primary-color});
+$btn-secondary-bg-color:var(#{--cf-verification-secondary-btn-bg-color});
+$btn-secondary-text-color:var(#{--cf-verification-primary-color});
+$action-box-bg-color:var(#{--cf-verification-action-box-bg-color});
+$action-box-text-color:var(#{--cf-verification-action-box-text-color});
+$link-color:var(#{--cf-verification-primary-color});
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/settings/_variables.scss:15]: It doesn't make sense to put the CSS variables into SCSS variables. Just use the CSS variables directly where you need them. I'm not even sure if it works like that.

File: packages/verification-vue-component/src/components/Verification/VerificationItem.vue [hunk 11]
```diff
@@ -3,7 +3,7 @@
          :class="{ 'confirmation-step': digitalTwin.confirmationStep, 'error': digitalTwinInformation.error }">
         <div class="card-container">
             <div v-if="digitalTwinInformation.active" class="header">
-                <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                <img :src="headerLogo" alt="logo"/>
```
Comment (bauer00) [inline packages/verification-vue-component/src/components/Verification/VerificationItem.vue:6]: Have you tried switching to CSS for the logo? With CSS it would be at one place to change the theme, not multiple.

File: packages/verification-vue-component/src/assets/img/hin-logo.svg [hunk 12]
```diff
@@ -0,0 +1,9 @@
+<svg width="67" height="28" viewBox="0 0 67 28" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (bauer00) [inline packages/verification-vue-component/src/assets/img/hin-logo.svg:1]: This image should be removed from the package.

File: packages/verification-vue-component/package.json [hunk 13]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@certifaction/verification-vue-component",
-    "version": "3.3.16",
+    "version": "3.3.17",
```
Comment (bauer00) [inline packages/verification-vue-component/package.json:3]: Why this change? This should only be changed when a new release is published with lerna.

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 14]
```diff
@@ -1,3 +1,19 @@
+
+:root {
+    --cf-verification-primary-color: #0669f2;
+    --cf-verification-secondary-btn-bg-color: #e5e3d6;
+    --cf-verification-default-btn-bg-color: #ffffff ;
+    --cf-verification-action-box-bg-color: #e6f0fe ;
+    --cf-verification-action-box-text-color: #0d6efd ;
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/settings/_variables.scss:7]:
```suggestion
    --cf-verification-default-btn-bg-color: #ffffff;
    --cf-verification-action-box-bg-color: #e6f0fe;
    --cf-verification-action-box-text-color: #0d6efd;
```

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 15]
```diff
@@ -1,3 +1,19 @@
+
+:root {
+    --cf-verification-primary-color: #0669f2;
+    --cf-verification-secondary-btn-bg-color: #e5e3d6;
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/settings/_variables.scss:4]: Inconsistent naming, one is just called primary-color and one secondary-**btn-bg**-color, why not just secondary-color?

File: packages/verification-vue-component/src/components/Verification/VerificationItem.vue (hunk 11 above)
Comment (bauer00) [reply packages/verification-vue-component/src/components/Verification/VerificationItem.vue:6]: What did you try? I would expect this would work:
```
:root {
    --cf-verification-logo-url: url('path/to/logo.svg');
}

background-image: var(--cf-verification-logo-url);
```
Maybe it would also make sense to specify the `width` and `height` with CSS variables.

File: packages/verification-vue-component/src/style/components/verification/_item_list.scss [hunk 16]
```diff
@@ -105,9 +106,12 @@
                 height: 100%;
 
                 > .header {
-                    height: 65px;
+                    height: 78px;
                     padding: 20px;
                     background-color: $white;
+                    background-repeat: no-repeat;
+                    background-position: 12px 5px;
+                    background-image: var(--cf-verification-logo-url);
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/components/verification/_item_list.scss:114]: The background size should be defined, probably the height fix and width auto to make sure the passed logo always fits.

File: packages/verification-vue-component/src/style/settings/_variables.scss [hunk 17]
```diff
@@ -1,3 +1,13 @@
+
+:root {
+    --cf-verification-primary-color:#0669f2;
+    --cf-verification-secondary-color:#e5e3d6;
+    --cf-verification-default-btn-bg-color:#ffffff;
+    --cf-verification-action-box-bg-color:#e6f0fe;
+    --cf-verification-action-box-text-color:#0d6efd;
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/settings/_variables.scss:7]: No space after the `:`

File: packages/verification-vue-component/src/style/components/verification/_item_list.scss [hunk 18]
```diff
@@ -105,9 +106,12 @@
                 height: 100%;
 
                 > .header {
-                    height: 65px;
+                    height: 78px;
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/components/verification/_item_list.scss:109]: Why is this needed? Doesn't this change the overall layout?

File: .gitignore (hunk 2 above)
Comment (bauer00) [reply .gitignore:14]: I agree with Alessandro here, these files shouldn't be ignored.

File: packages/verification-app/.env (hunk 9 above)
Comment (bauer00) [reply packages/verification-app/.env:10]: Alessandro updated the verification package to the latest version, so please rebase or merge the latest changes into this branch.

File: packages/verification-vue-component/package.json [hunk 19]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@certifaction/verification-vue-component",
-    "version": "3.3.17",
+    "version": "3.3.16",
```
Comment (bauer00) [inline packages/verification-vue-component/package.json:3]: I think this is because of the rebase, it needs to be 3.3.17 now of course.

**Review by bauer00 — COMMENTED** [review]
I will also commit the proper lock files, as you probably used normal npm to install the packages. Here we used yarn as package manager as it worked better together with lerna.

File: packages/verification-vue-component/src/style/components/verification/_item_list.scss [hunk 20]
```diff
@@ -108,6 +109,10 @@
                     height: 65px;
                     padding: 20px;
                     background-color: $white;
+                    background-repeat: no-repeat;
+                    background-position: 12px 5px;
+                    background-size: contain;background-size: contain;
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/components/verification/_item_list.scss:114]: Double

File: packages/verification-vue-component/src/style/components/verification/_item_list.scss [hunk 21]
```diff
@@ -108,6 +109,10 @@
...
+                    background-image: var(--cf-verification-logo-url);
 
                     img {
```
Comment (bauer00) [inline packages/verification-vue-component/src/style/components/verification/_item_list.scss:117]: This is not needed anymore as the logo is now a background image.

**Review by bauer00 — APPROVED** [review]
Rest looks good
